### ISSUE TITLE
TensorBoard with Histograms crashes with Dropout Layers

### ISSUE THREAD

**Issue by jdoerrie** [fchollet/keras#2297]
The most recent version of Keras crashes after the first epoch when a TensorBoard callback is added that records histograms.

Failing Example Code (`examples/mnist_mlp.py` with added callback) and Crash Log:
https://gist.github.com/jdoerrie/713a2d53e14b01470aa664494186d7c7

Judging by the output the issue is related to `keras_learning_phase` not being fed. The code works normally when `histogram_freq` is set to 0 or the Dropout layers are removed.

- Operating System: Mac OS X 10.11.3
- Python: 3.5.1
- Keras: 1.0.0 (Master Checkout)
- Tensorflow: 0.7.1
- Running on CPU.

**Comment by tboquet** [comment]
I'm on it, I will submit a pull request in a few minutes.

**Comment by jcomfort4** [comment]
I just received this same error when running on Windows 10 w/ tensorflow-gpu -- I had set the following callbacks..

`tensorboard = TensorBoard(log_dir='./logs', histogram_freq=1, write_graph=True, write_images=True, embeddings_freq=1)`

The error I received was the same as the one listed on this thread where hte issue was with keras_learning_phase

Based on this bug, I set histogram_freq and embeddings_freq to 0 and tried again and everything worked successfully.
